lib/node_modules/@stdlib/math/base/special/log10/examples/index.js

```diff
@@ -18,14 +18,13 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var logEachMap = require( '@stdlib/console/log-each-map' );
 var log10 = require( './../lib' );
 
-var x;
-var i;
+var opts = {
+	'dtype': 'float64'
+};
+var x = discreteUniform( 100, 0, 100, opts );
 
-for ( i = 0; i < 100; i++ ) {
-	x = round( randu() * 100.0 );
-	console.log( 'log10(%d) = %d', x, log10( x ) );
-}
+logEachMap( 'log10(%0.4f) = %0.4f', x, log10 );
```

Comment:
@stdlib-bot This should become 

`logEachMap( 'log10(%d) = %0.4f', x, log10 );`

since we don't want to display decimal points for integers.